src/drivers/pwm_out_sim/PWMSim.hpp

```diff
@@ -63,6 +63,8 @@ class PWMSim : public cdev::CDev, public ModuleBase<PWMSim>, public OutputModule
 	/** @see ModuleBase::print_status() */
 	int print_status() override;
 
+	const char *get_param_prefix() override { return "PWM_SIM"; }
```

Comment:
@dagar proposed making this `PWM_MAIN`, since it will be the only output module in use for SITL.  I'll make that change and remove all the `PWM_SIM*` params.